en/docs/cookies/oneflow2onnnx.md

```diff
@@ -0,0 +1,209 @@
+
+2. [onnxruntime & onnxruntime-gpu](https://github.com/microsoft/onnxruntime): ONNX runtime that is used to load the ONNX model for inference. onnxruntime and onnxruntime-gpu support CPU inference and GPU inference respectively.
+
+3. [onnx-simplifier](https://github.com/daquexian/onnx-simplifier): structure for simplifying ONNX models, e.g. eliminating operators with constant results
+   
+4. [onnxoptimizer](https://github.com/onnx/optimizer): it is used to optimize ONNX model through graph transformations
+
+
+## Export OneFlow Models to ONNX Models
+
+[oneflow-onnx](https://github.com/Oneflow-Inc/oneflow_convert) is a model conversion tool provided by OneFlow team to support exporting OneFlow static diagram models to ONNX models. At present oneflow-onnx supports more than 80 kinds of Oneflow OP exported as ONNX OP. For detalis, refer to [list of OP supported by OneFlow2ONNX](https://github.com/Oneflow-Inc/oneflow_convert/blob/main/docs/oneflow2onnx/op_list.md)。
+
+### Install oneflow-onnx
+
+oneflow-onnx is independent of OneFlow and needs to be installed separately via pip:
+```bash
+pip install oneflow-onnx
+```
+
+### How to Use oneflow-onnx
+
+To export OneFlow static graph model as ONNX model, just call ` export_ onnx_ Model ` function.
+
+```python
+from oneflow_onnx.oneflow2onnx.util import export_onnx_model
+
+export_onnx_model(graph,
+                  external_data=False, 
+                  opset=None, 
+                  flow_weight_dir=None, 
+                  onnx_model_path="/tmp", 
+                  dynamic_batch_size=False)
+```
+The meaning of each parameter is as follows:
+
+1. graph: the graph need to be converted ( [Graph](../basics/08_nn_graph.md) object )
+
+2. external_data: whether to save the weights as external data of the ONNX model. When it is `True`, it is usually to avoid the 2GB file size limit of protobuf.
+
+3. opset: specify the version of the conversion model ( int, default is 10 )
+
+4. flow_weight_dir: path to save OneFlow model weights
+
+5. onnx_model_path: save path for exported ONNX models
+
+6. dynamic_batch_size: whether the exported ONNX model supports dynamic batch, default is False
+
+
+In addition, oneflow-onnx provides a function called `convert_to_onnx_and_check` to convert and check the converted ONNX model. The check refers to feeding the same input to the original OneFlow model and the converted ONNX model respectively, and then comparing the difference between each value in the two outputs to see if it is within a reasonable margin of error.
+
+```python
+from oneflow_onnx.oneflow2onnx.util import convert_to_onnx_and_check
+
+convert_to_onnx_and_check(...)
+```
+The parameters of the `convert_to_onnx_and_check` are a superset of the parameters of the `export_onnx_model`, and you can be pass in `print_outlier=True` additionally to output any abnormal values found during the check process that exceed the reasonable error range. 
```

Comment:
```suggestion
The parameters of the `convert_to_onnx_and_check` are almost the same as those of `export_onnx_model`, besides  you can pass `print_outlier` parameter additionally. When `print_outlier=True`, it will output any abnormal values found during the check process that exceed the reasonable error range. 
```